crates/uv/tests/cache_prune.rs

```diff
@@ -145,25 +129,26 @@ fn prune_stale_symlink() -> Result<()> {
     let wheels = context.cache_dir.child("wheels-v0");
     fs_err::remove_dir_all(wheels)?;
 
-    let filters = [
-        (r"Pruning cache at: .*", "Pruning cache at: [CACHE_DIR]"),
-        (
-            r"Removing dangling cache entry: .*[\\|/]archive-v0[\\|/].*",
-            "Pruning cache at: [CACHE_DIR]/archive-v0/anyio",
-        ),
-    ]
-    .into_iter()
-    .chain(INSTA_FILTERS.to_vec())
-    .collect::<Vec<_>>();
+    let filters: Vec<_> = context
+        .filters()
+        .into_iter()
+        .chain([
+            // The cache entry does not have a stable key, so we filter it out
+            (
+                r"\[CACHE_DIR\](\\|\/)(.+)(\\|\/).*",
+                "[CACHE_DIR]/$2/[ENTRY]",
+            ),
+        ])
+        .collect();
```

Comment:
This is one of the two recommended ways to extend the standard filters. I'd generally recommend this. Alternatively, you may want to perform filtering _before_ the context filters in which case you'd just swap the chain.